docs/quickstart/quickstart_staged_tree_notebook.ipynb

```diff
@@ -0,0 +1,193 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Creation of a Staged Tree"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## EventTree Class\n",
+    "\n",
+    "The first starting point in constructing a Chain Event Graph (CEG) is to create \n",
+    "an *event tree* describing the process being studies. An event tree is a directed \n",
+    "tree graph with a single root node. The nodes with no emanating edges are\n",
+    "called *leaves*, and the non-leaf nodes are called *situations*. \n",
+    "\n",
+    "To create an event tree from a tabular data set in cegpy we\n",
+    "first need to import the data set and then initialise the \n",
+    "`EventTree` object which can be done with the following code."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cegpy import EventTree\n",
+    "import pandas as pd\n",
+    "\n",
+    "df = pd.read_excel('../../data/medical_dm_modified.xlsx')\n",
+    "et = EventTree(df)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to display the EventTree, we can use the method `create_figure()`. The numbers above the edges of the event tree represent the number of indiviuals who passed through the given edge. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "et.create_figure()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## StagedTree Class\n",
+    "\n",
+    "In an event tree, each situation is associated with a transition parameter vector which indicates the conditional\n",
+    "probability of an individual, who has arrived at the situation, going along one of its edges. In order to create a CEG, we first need to elict a *staged tree*. \n",
+    "This is done by first need to partition situations into *stages*, which are collection of situation in the event tree whose immediate evolutions, i.e. their\n",
+    "associated conditional transition parameter vectors, are equivalent. To indicate this symmetry, all situations in the same stage are assinged a single colour.\n",
+    "\n",
+    "Identification of the stages in the event tree can be done using any suitable model selection algorithm. Currently, the only avaible selection algoirhtm in `cegpy` is the *agglomerative hierarchical clustering (AHC)* algorithm (Freeman and Smith, 2011).\n",
+    "\n",
+    "In order to creat a staged tree in `cegpy` we first initialize a `StagedTree` object from the dataset and then run th AHC algorithm using the `create_AHC_tarnsitions` method, as displayed below. The output of the AHC algorithm is a dictionary containing the following information:\n",
```

Comment:
Minor spelling: "th AHC".

When I was trying to get the docs to build, I went through and cleaned up some other spelling mistakes. I missed this one.

FYI I use "Code Spell Checker" in vscode (Identifier: streetsidesoftware.code-spell-checker) and you may also find this useful.

Reply:
Thanks! I was meant to add spell checking to my workspace ages ago :) that is very helpful.